torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1379,7 +1379,6 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
             // Check every input's shape against profiled (expected) shape.
             for (i = 0; i < num_inputs; i++) {
               auto& input = peek(stack, i, num_inputs);
-              TORCH_INTERNAL_ASSERT(input.isTensor());
```

Comment:
this isnt really necessary to do since `toTensor()` has the same assertion - since its a one line change i grouped it in here